src/context/QuizContext/context/QuizProvider.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { useContext, useReducer } from "react";
+import { initialState, quizReducer } from "../reducer/quizReducer";
+import { QuizContextType } from "./QuizContext.type";
+
+const QuizContext = React.createContext({} as QuizContextType);
+
+export const QuizContextProvider: React.FC = ({ children }) => {
+  const [quizState, quizDispatch] = useReducer(quizReducer, initialState);
+  return <QuizContext.Provider value={{ quizState, quizDispatch }}>{children}</QuizContext.Provider>;
```

Comment:
You can format this return line into 2 lines for code to look clean

Reply:
this was due to my prettier format width settings